### ISSUE TITLE
Escape doesn't escape anything at all when using double quotes

### ISSUE THREAD

**Issue by kornelski** [db-migrate/db-migrate-base#5]
https://github.com/db-migrate/db-migrate-base/blob/05762f395e43643c2e9a1641e7b2a9492f34ae5a/index.js#L682

```js
return str.replace(/"/g, '"');
```

The replacement string was probably meant to be `'\\"'`.

The code should be

```js
return str.replace(/\\/g, '\\\\').replace(/"/g, '\\"');
```

**Comment by wzrdtales** [comment]
@pornel Yes you're right, that wont work quite well though. Thanks for point out :) Feel free to open a PR if you want to, I will have a look at this later on though.
